src/test/java/de/tum/in/www1/artemis/authentication/PersonalAccessTokensTest.java

```diff
@@ -0,0 +1,73 @@
+package de.tum.in.www1.artemis.authentication;
+
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.http.HttpStatus;
+import org.springframework.security.test.context.support.WithAnonymousUser;
+import org.springframework.security.test.context.support.WithMockUser;
+
+import de.tum.in.www1.artemis.AbstractSpringIntegrationJenkinsGitlabTest;
+import de.tum.in.www1.artemis.domain.User;
+import de.tum.in.www1.artemis.repository.UserRepository;
+
+public class PersonalAccessTokensTest extends AbstractSpringIntegrationJenkinsGitlabTest {
```

Comment:
You've got some Codacy issues here

Reply:
You are right. We'll fix that. Nevertheless, we need some more reviews from @ls1intum/artemis-maintainers or @ls1intum/artemis-developers I guess :)

Reply:
Still there 😉

Reply:
Thanks :) Hopefully fixed now!